package.json

```diff
@@ -20,7 +20,6 @@
     "release": "standard-version && yarn build && git push --follow-tags && npm publish"
   },
   "files": [
-    "src",
```

Comment:
Removing this is breaking change for 2.x (#51)

Reply:
So should I remove it?

Reply:
Never mind I removed it, the critical fix here is not related to it